packages/lit-html/package.json

```diff
@@ -6,41 +6,54 @@
   "homepage": "https://lit-html.polymer-project.org/",
   "repository": "Polymer/lit-html",
   "type": "module",
-  "main": "index.js",
-  "typings": "index.d.ts",
+  "main": "lit-html.js",
+  "typings": "lit-html.d.ts",
   "directories": {
     "test": "test"
   },
   "files": [
-    "/index.js",
-    "/index.js.map",
-    "/index.d.ts",
-    "/index.d.ts.map",
-    "/lib/",
+    "/lit-html.js",
+    "/lit-html.js.map",
+    "/lit-html.d.ts",
+    "/lit-html.d.ts.map",
     "/directives/",
     "/src/",
-    "!/src/test/"
+    "!/src/test/",
+    "/development/",
+    "!/development/test/"
   ],
   "scripts": {
-    "build": "tsc",
+    "build": "npm run clean && tsc && rollup -c",
```

Comment:
I'd like to keep a quick-build script around. Maybe as `build:ts`?

Reply:
Ok, I added `build:ts` and also renamed `build:watch` to `build:ts:watch` to make it more clear which step it is.